src/NuGet.Core/NuGet.ProjectModel/PackageSpecOperations.cs

```diff
@@ -142,24 +164,21 @@ private static List<LibraryDependency> GetExistingDependencies(PackageSpec spec,
             }
             else
             {
-                var dependency = new LibraryDependency
-                {
-                    LibraryRange = new LibraryRange(packageId, range, LibraryDependencyTarget.Package)
-                };
-
-                list.Add(dependency);
+                AddDependency(spec, list, packageId, range);
             }
 
         }
 
         private static void AddDependency(
+            PackageSpec spec,
```

Comment:
+1, but I wouldn't call it a nitpick. I was writing a comment elsewhere suggesting the `list` could be removed from all the methods, because seeing `(spec, spec.Dependencies, ...)` at the call's entry point looks weird. However, if there's only a single property used, then I agree we shouldn't pass in the whole data structure.